fusor-ember-cli/app/mirage/config.js

```diff
@@ -14,7 +14,7 @@ export default function() {
   this.del('/fusor/api/v21/deployments/:id');
   this.get('/fusor/api/v21/deployments/:id/openshift_disk_space',
     function(db, request) {
-      return { openshift_disk_space: 1024 };
+      return { openshift_disk_space: 1024 * 250 };
```

Comment:
Now it shows 250 GB (and no error) rather than 1 GB (with error) when using `ember s --environment mocks-enabled`